go/vt/sqlparser/ast_format.go

```diff
@@ -1314,12 +1314,10 @@ func (node *Literal) Format(buf *TrackedBuffer) {
 	switch node.Type {
 	case StrVal:
 		sqltypes.MakeTrusted(sqltypes.VarBinary, node.Bytes()).EncodeSQL(buf)
-	case IntVal, FloatVal, DecimalVal, HexNum:
+	case IntVal, FloatVal, DecimalVal, HexNum, BitNum:
```

Comment:
I wonder, should we also refactor `HexVal` away? Or is there a legitimate reason we need to distinguish `HexNum` and `HexVal`? And if so, wouldn't that also apply to the `Bit` literals?